### ISSUE TITLE
better parsing for author order

### ISSUE THREAD

**Issue by tklebel** [tklebel/jstor#23]
use "contrib-id", if available

**Comment by starship9** [comment]
I'd like to take this, but the issue seems a bit vague. Could you elaborate on it a bit?

**Comment by tklebel** [comment]
Yep, will do. The vagueness was the reason for removing the labels again a few days ago, but it should be possible to tackle this issue, once I add a bit more info. I will ping you when I updated the issue with more information.

**Comment by maelle** [comment]
`jstor` was archived in 2026 because the data source for obtaining records which could be processed with `jstor` became unavailable in 2021. 
 
 See #91
